presto-spi/src/test/java/com/facebook/presto/spi/type/TestUnscaledDecimal128Arithmetic.java

```diff
@@ -107,6 +108,18 @@ public void testRescaleOverflows()
     }
 
     @Test
+    public void testAdd()
```

Comment:
comprehensive tests are in `TestDecimalOperators`. Here are only basic test cases which are remains of early iterations of TDD